src/robinhood.js

```diff
@@ -60,6 +60,7 @@ function RobinhoodWebApi(opts, callback) {
         news: 'midlands/news/',
         tag: 'midlands/tags/tag/'
     },
+    _clientId = 'c82SH0WZOsabOXGP2sxqcj34FxkvfnWRZBKlBjFS',
```

Comment:
Where do you get this?

Reply:
I got it from the official Robinhood web app.  Wondering if 1) its the same for everybody and 2) its okay to use here

Reply:
@chiefsmurph were you able to auth w/ the your code in this PR?

Reply:
I was able to authenticate with these changes. As for _clientId, I'm getting the same value in Chrome DevTools:Network on Robinhood Web, so it would appear to be the same for everyone.